website/docs/devDocs/Development.md

```diff
@@ -146,3 +146,33 @@ Checkstyle plugin may help to detect violations directly from the IDE.
    ```bash
    helm upgrade --set submarine.database.dev=true submarine ./helm-charts/submarine
    ```
+
+## Develop Submarine Website
+Submarine website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
+
+We store all the website content in markdown format in the `submarine/website/docs`. When committing a new patch to the `submarine` repo, Docusaurus will help us generate the `html` and `javascript` files and push them to  https://github.com/apache/submarine-site/tree/asf-site.
+
+If you want to update the website, you can click “Edit this page” on the website.
+
+![](https://lh4.googleusercontent.com/gYcKpxbsGAKv2giTRqkxOehPGnuvnhE31WjsAsYhFmACIZF3Wh2ipar7mZ7F_KRwecM-L1J8YJAgNigJsJUjqc-5IXeO2XGxCIcYpP9CdSc3YByuUkjT_Bezby2HHtkBLyE1ZY_F)
```

Comment:
is this an empty ink?

Reply:
It shows the image on the website.
Like below.
![image](https://user-images.githubusercontent.com/37936015/125725773-56bd9296-250f-4470-a323-9dac3173071f.png)